src/verify/VerificationCodeInputScreen.tsx

```diff
@@ -94,25 +103,39 @@ function VerificationCodeInputScreen({
 
   return (
     <SafeAreaView style={styles.container} edges={['bottom']}>
-      <KeyboardAwareScrollView
-        style={[styles.scrollContainer, headerHeight ? { marginTop: headerHeight } : undefined]}
-        keyboardShouldPersistTaps="always"
+      <KeyboardAvoidingView
```

Comment:
all these changes are just to enable the keyboard to shrink the screen content so that the resend button is above the keyboard....

Reply:
ah, thanks for the reminder! i had forgotten about the keyboard spacer component. it works well like you suggested, i also applied a similar change to the phone number screen now